README.md

```diff
@@ -14,56 +14,65 @@ As this guide is an extension of the [Christopher Adams Elixir Style Guide](http
 # Table of Contents
 
 * __[General Guidelines](#general-guidelines)__
-  * [Breaking Lines](#breaking-lines)
+  * [Aliases](#aliases)
   * [Blank Lines](#blank-lines)
+  * [Breaking Lines](#breaking-lines)
   * [Dangling Commas](#dangling-commas)
-  * [Aliases](#aliases)
+  * [Moduledocs](#moduledocs)
   * ['With do' Indentation](#with-do-indentation)
 * __[Schemas](#schemas)__
   * [Ecto Schema](##ecto-schema)
   * [GraphQL Schema](##graphql-schema)
```

Comment:
These title links are not working. Probably the problem is the term in the parenthesis that contains 2 `#`.